cassandra/src/test/java/org/hibernate/ogm/datastore/cassandra/test/embeddable/EmbeddableMappingTest.java

```diff
@@ -0,0 +1,202 @@
+				.assertColumn( "homeAddress.type.name", "main" )
+				.assertColumn( "postal_code", "75007" )
+				.assertColumn( "password", "like I would tell ya" )
+				.assertColumn( "version", 0 )
+				.assertNoOtherColumnPresent()
+				.execute();
+
+		transaction.commit();
+		session.clear();
+
+		transaction = session.beginTransaction();
+
+		Account loadedAccount = session.get( Account.class, account.getLogin() );
+
+		// When
+
+		// set some values to null
+		loadedAccount.getHomeAddress().setCountry( null );
+		loadedAccount.setPassword( null );
+		session.merge( loadedAccount );
+
+		transaction.commit();
+
+		transaction = session.beginTransaction();
+
+		// Then
+		rowAssertion( session.getSessionFactory(), "Account" )
+				.keyColumn( "login", "emmanuel" )
+				.assertColumn( "login", "emmanuel" )
+				.assertColumn( "homeAddress.city", "Paris" )
+				.assertColumn( "homeAddress.street1", "1 avenue des Champs Elysees" )
+				.assertColumn( "homeAddress.type.name", "main" )
+				.assertColumn( "postal_code", "75007" )
+				.assertColumn( "version", 1 )
+				.assertNoOtherColumnPresent()
+				.execute();
+
+		transaction.commit();
+
+		session.clear();
+
+		transaction = session.beginTransaction();
+
+		loadedAccount = session.get( Account.class, account.getLogin() );
+
+		// When
+		// set a nested embedded to null
+		loadedAccount.getHomeAddress().setType( null );
+		session.merge( loadedAccount );
+
+		transaction.commit();
+
+		transaction = session.beginTransaction();
+
+		// Then
+		rowAssertion( session.getSessionFactory(), "Account" )
+				.keyColumn( "login", "emmanuel" )
+				.assertColumn( "homeAddress.city", "Paris" )
+				.assertColumn( "homeAddress.street1", "1 avenue des Champs Elysees" )
+				.assertColumn( "postal_code", "75007" )
+				.assertColumn( "version", 2 )
+				.assertNoOtherColumnPresent()
+				.execute();
+
+		transaction.commit();
+
+		session.clear();
+
+		transaction = session.beginTransaction();
+
+		loadedAccount = session.get( Account.class, account.getLogin() );
+
+		// When
+		// set all properties of an embedded to null
+		loadedAccount.getHomeAddress().setCity( null );
+		loadedAccount.getHomeAddress().setStreet1( null );
+		session.merge( loadedAccount );
+
+		transaction.commit();
+
+		transaction = session.beginTransaction();
+
+		// Then
+		rowAssertion( session.getSessionFactory(), "Account" )
+				.keyColumn( "login", "emmanuel" )
+				.assertColumn( "login", "emmanuel" )
+				.assertColumn( "postal_code", "75007" )
+				.assertColumn( "version", 3 )
+				.assertNoOtherColumnPresent()
+				.execute();
+
+		transaction.commit();
+		// Clean-Up
+		transaction = session.beginTransaction();
+		loadedAccount = session.get( Account.class, account.getLogin() );
+		session.delete( loadedAccount );
+		transaction.commit();
+
+		session.close();
+	}
+
+	@Test
+	@Ignore("@ElementCollection seems not to work in Cassandra if no @OrderColumn is specified")
```

Comment:
Indeed that's not supported, as we cannot determine a primary key for the table in such case. I'll remove the test method altogether. Once we have interfaces for the mapping tests in core (see [OGM-965](https://hibernate.atlassian.net/browse/OGM-965), we'd have to skip the test on Cassandra via `@SkipByGridDialect`.